server/jetstream_cluster.go

```diff
@@ -3861,15 +3875,21 @@ func (js *jetStream) monitorConsumer(o *consumer, ca *consumerAssignment) {
 
 			currentCount, firstPeer, foundLeader := currentPeerCount(ci, newPeerSet, selfId)
 
-			// If all are current we are good, or if we have some offline and we have a quorum.
-			if quorum := replicas/2 + 1; currentCount >= quorum {
+			// If all are current we are good
+			if currentCount == replicas {
+				if mDelayTc < scaleDownDelayTicks {
+					mDelayTc++
+					continue
+				}
 				// Remove the old peers or transfer leadership (after which new leader resumes with peer removal).
 				// stopMigrationMonitoring is invoked on actual leadership change or
 				// on the next tick when migration completed.
 				// In case these operations fail, the next tick will retry
 				if !foundLeader {
+					s.Debugf("Scale down of '%s > %s' step down ('%s' preferred)", ca.Stream, ca.Name, firstPeer)
 					n.StepDown(firstPeer)
 				} else {
+					s.Noticef("Scale down of '%s > %s' to %+v", ca.Stream, ca.Name, newPeerSet)
```

Comment:
Should be notice? Every time we log?

Reply:
I find this to be a rather significant event, worth of a log entry. But not set on this. @derekcollison WDYT?

The whole process starts with this notice:
	s.Noticef("Requested move of: R=%d stream '%s > %s' from old peer set %+v to new peer set %+v",
		cfg.Replicas, streamName, accName, s.peerSetToNames(currPeers), s.peerSetToNames(peers))

Reply:
ok good to bookend and keep consistent.